.github/actions/download-plugin/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
     description: 'The version of security plugin that should be used, e.g "3.0.0.0"'
     required: true
 
+  download-location:
+    description: 'The location of where to download the plugin'
+    required: true
```

Comment:
Agree, but I ran into a lot of issues with github runners and relative paths, deciding to keep it the same for now until there are more usages/we decide it is cleaner. Since the tests pass, I think we are good for now.